src/assets/images/triangles-combined-earth-raise-white.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="53" height="39" viewBox="0 0 53 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M38.7009 13.6572L52.3535 38.6959H25.0386L38.7009 13.6572Z" fill="#fff"/>
+    <path d="M20.4789 36.4199L6.36785e-06 5.89713e-05L40.9724 6.61352e-05L20.4789 36.4199Z" fill="#50B400"/>
```

Comment:
matches existing colour https://github.com/thebiggive/components/blob/c096dcaa499ece7080a363d4157bc2e1cd1c5c92/src/globals/variables.scss#L25 and https://github.com/thebiggive/components/blob/c096dcaa499ece7080a363d4157bc2e1cd1c5c92/src/globals/variables.scss#L26